# GBM
The scripts for article "Multi-omics analysis characterizes the spatial architecture of glioblastoma ecoregions".

The version of the software used in the analysis:

Analysis of electrophysiological recordings: Clampfit 10.3 software.

Image processing: ImageJ (v2.1.0), Stardist (v0.8.3).

Preprocessing of sequencing data: CellRanger(v6.0.2), SpaceRanger(v1.2.1), APEC (v1.1.0), STAR (v2.5.3), and RSEM (v1.3.1).

scRNA-seq data processing: scDblFinder (v1.7.7), Seurat (v4.3.0), Scanpy (v1.9.3), souporcell (v2.4), inferCNV (v1.6.0), Palantir (v1.3.3), and PHATE (v1.0.11). 

Deep learning model predict cell types: scArches (v0.5.9)

Genomic data processing: BWA-MEM (c0.7.17), GATK (v4.1.3).

Spatial transcriptome related analysis: Cell2location (v0.6), SPACEL (v1.1.6), COMMOT (v0.0.3), Tangram (v1.0.2), and scVelo (v0.3.2).

ISH data processing: ASHLAR (v1.14.1), DeepCell (v0.11.0) and Starfish (v0.2.2).

scATAC processing: Signac (v1.9.0), Harmony (v1.0), and ArchR (v1.0.1).

Pre-trained models required for transfer spatial domains by SCANVI: https://doi.org/10.5281/zenodo.15699049
